tests/run_model_templates_tests.py

```diff
@@ -274,6 +277,27 @@ def is_in_virtual_env_in_work_dir(work_dir, domain):
 def generate_venv_path(work_dir, domain):
     return os.path.join(work_dir, domain, VENV_FOLDER_NAME)
 
+def pytest_run_tests(all_tests, work_dir, verbose):
+    pytest_ids = []
+    for el in all_tests:
+        curfile = inspect.getfile(type(el['test']))
+        curfile = os.path.relpath(curfile)
+        classname = type(el['test']).__name__
+        cur_id = el['id']
+        cur_id = cur_id[cur_id.rfind('.')+1:]
+        cur_pytest_id = f'{curfile}::{classname}::{cur_id}'
+        pytest_ids.append(cur_pytest_id)
+
+    pytest_ids = ' '.join(pytest_ids)
+    verb_flag = '-v' if verbose else ''
```

Comment:
kind of useless

Reply:
Why? a user can run the script without option '--verbose', in this case pytest will print much less info.

Reply:
nevermind